test/FunctionalTests/Tests/DataOData/Tests/OData.Writer.Tests/JsonLight/JsonLightPropertyWriterTests.WriteUntypedValueTest.approved.txt

```diff
@@ -110,3 +110,51 @@ Combination: 32; TestConfiguration = Format: JsonLight, Request: False, Synchron
 Model Present: true
 {"@odata.context":"http://odata.org/test/$metadata#Collection(TestModel.JsonType)","value":["foo bar",[1, 2, "abc"],3.1415]}
 
+Combination: 33; TestConfiguration = Format: JsonLight, Request: True, Synchronous: True
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.Int32","value":42}
+
+Combination: 34; TestConfiguration = Format: JsonLight, Request: False, Synchronous: True
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.Int32","value":42}
+
+Combination: 35; TestConfiguration = Format: JsonLight, Request: True, Synchronous: False
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.Int32","value":42}
+
+Combination: 36; TestConfiguration = Format: JsonLight, Request: False, Synchronous: False
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.Int32","value":42}
+
+Combination: 37; TestConfiguration = Format: JsonLight, Request: True, Synchronous: True
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.Double","value":3.1415}
+
+Combination: 38; TestConfiguration = Format: JsonLight, Request: False, Synchronous: True
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.Double","value":3.1415}
+
+Combination: 39; TestConfiguration = Format: JsonLight, Request: True, Synchronous: False
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.Double","value":3.1415}
+
+Combination: 40; TestConfiguration = Format: JsonLight, Request: False, Synchronous: False
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.Double","value":3.1415}
+
+Combination: 41; TestConfiguration = Format: JsonLight, Request: True, Synchronous: True
+Model Present: true
+{"@odata.context":"http://odata.org/test/$metadata#Edm.String","value":"foo bar"}

```

Comment:
Yes. I used it because it was consistent with the rest of the JSON light property writer tests, but will replace it throughout.

---
In reply to: [128104508](https://github.com/OData/odata.net/pull/894#discussion_r128104508) [](ancestors = 128104508)